packages/gui/src/components/nfts/gallery/NFTGallery.tsx

```diff
@@ -253,6 +254,21 @@ export default function NFTGallery() {
     };
   }, []);
 
+  React.useEffect(() => {
+    if (allowNFTsFiltered.length) {
+      if (document.getElementById('scroll-helper') && scrollPosition.get('position') > 0) {
+        setTimeout(() => {
+          if (document.getElementById('scroll-helper')?.parentNode) {
+            (document.getElementById('scroll-helper')?.parentNode as HTMLElement).scrollTo(
```

Comment:
Currently this is setting the scroll position whenever the NFTGallery is mounted. I was hoping that the scroll position would only be restored when navigating back from NFTDetails.